sdk/impactreporting/Azure.ResourceManager.ImpactReporting/api/Azure.ResourceManager.ImpactReporting.net8.0.cs

```diff
@@ -0,0 +1,616 @@
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.ClientIncidentDetails>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.ClientIncidentDetails>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ConfidenceLevel : System.IEquatable<Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ConfidenceLevel(string value) { throw null; }
+        public static Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel High { get { throw null; } }
+        public static Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel Low { get { throw null; } }
+        public static Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel Medium { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel left, Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel right) { throw null; }
+        public static implicit operator Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel left, Azure.ResourceManager.ImpactReporting.Models.ConfidenceLevel right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class Connectivity : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ImpactReporting.Models.Connectivity>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ImpactReporting.Models.Connectivity>
+    {
+        public Connectivity() { }
+        public int? Port { get { throw null; } set { } }
+        public Azure.ResourceManager.ImpactReporting.Models.Protocol? Protocol { get { throw null; } set { } }
+        public string SourceAzureResourceId { get { throw null; } set { } }
+        public string TargetAzureResourceId { get { throw null; } set { } }
```

Comment:
I think these properties are "arm resource id"s, right?
If so, you might need to change the property type in your typespec file to `armResourceIdentifier`.
This will not change the payload (it would still be a string), and it would not impact other SDKs (they treat armResourceIdentifier the same as a string).